dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/AddWebHook.cshtml

```diff
@@ -13,5 +13,14 @@
         <govuk-checkboxes-item value="@true">Enabled</govuk-checkboxes-item>
     </govuk-checkboxes>
 
+    <govuk-checkboxes asp-for="WebHookMessageTypes">
+        <govuk-checkboxes-fieldset>
+            <govuk-checkboxes-fieldset-legend class="govuk-fieldset__legend--m" />
+
+            <govuk-checkboxes-item value="@WebHookMessageTypes.UserUpdated" checked="@Model.WebHookMessageTypes.HasFlag(WebHookMessageTypes.UserUpdated)">User Updated</govuk-checkboxes-item>
```

Comment:
Maybe `UserUpdatedMessage.MessageTypeName` instead of 'User Updated' here? Having the exact message type name as is used in messages seems reasonable rather than a human-readable description.

Reply:
Done